src/tests/ftest/cart/test_group_np_srv.c

```diff
@@ -43,17 +42,13 @@ swim_crt_event_cb(d_rank_t rank, enum crt_event_source src,
 		"rank = %d, "
 		"crt_event_source = %d, "
 		"crt_event_type = %d\n",
-		 rank, src, type);
+		rank, src, type);
 
-	/* convert integer to string */
-	snprintf(type_to_a, 2, "%d", type);
+	swim_state_str[0] = type + '0';
+	swim_state_str[1] = 0;
```

Comment:
Is this suppose to be char "0" or is it suppose to be the end of strint "\0" ?

Reply:
It's supposed to be the end of string "\0".